lib/puppet/configurer.rb

```diff
@@ -161,6 +161,7 @@ def run_internal(options)
         begin
           if node = Puppet::Node.indirection.find(Puppet[:node_name_value],
               :environment => Puppet::Node::Environment.remote(@environment),
+              :configured_environment => Puppet[:environment],
```

Comment:
If the `environment` and `configured_environment` parameters are always the same, then I don't understand why we're doing this?